src/components/ProjectCard.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from "styled-components";
+import Img from "gatsby-image";
+
+const Card = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+  padding: 20px;
+  margin: 0 0.5em;
+  margin-bottom: 1em;
+`;
+
+const ProjectDescription = styled.p`
+  text-align: center;
+  font-style: italic;
+`;
+
+const ProjectPreview = styled(Img)`
+  border: 5px solid #555;
+`;
+
+import Button from "./Button";
+
+const ProjectCard = ({ project }) => {
+  const displayPreview = () => {
```

Comment:
I think there's probably a better way to do the conditional rendering on this preview image. Are you basically aiming to check whether a preview image exists, and rendering a message if not?

Could use a ternary operator which looks at `preview.childImageSharp.fixed` (which will be truthy if it exists but falsey if not)?

Reply:
I had a go at that but it wasn't working, may have related to the issues I was having with the query though so will try a ternary operator again

Reply:
Resolved this by checking whether project.frontmatter.preview exists, as opposed to the image itself.